src/status_im/ui/screens/chat/views.cljs

```diff
@@ -446,7 +585,13 @@
             mutual-contact-requests-enabled? @(re-frame/subscribe [:mutual-contact-requests/enabled?])
             max-bottom-space (max @bottom-space @panel-space)]
         [:<>
-         [topbar]
+         [topbar/topbar {:navigation      :none
```

Comment:
could you left a comment that it's better to do not use topbar component here, because of performance